tests/Geb/positive/test001.lisp

```diff
@@ -0,0 +1,49 @@
+(defpackage #:test001
```

Comment:
I don't think we should keep two or more versions of every test. There should be either test001.geb or test001.lisp, but not both. Maintaining them -- keeping in sync -- will become an unnecessary burden, and the only purpose of it is to check that we're parsing the lisp header correctly, which is not even strictly necessary.

Reply:
Done.